apps/sms/views/inbox/test/unit/inbox_test.js

```diff
@@ -667,6 +668,9 @@ suite('thread_list_ui', function() {
 
       assert.isFalse(firstThreadNode.classList.contains('unread'));
       assert.isFalse(secondThreadNode.classList.contains('unread'));
+
+      sinon.assert.neverCalledWith(MessageManager.markThreadRead, 1, true);
```

Comment:
@kumarrishav , I get this stack trace :- 
`Error: expected markThreadRead to be called with arguments 1, false
    markThreadRead(2, true)
    at fail (app://sms.gaiamobile.org/common/vendor/sinon/sinon.js:5406:29)
    at failAssertion (app://sms.gaiamobile.org/common/vendor/sinon/sinon.js:5367:13)
    at (anonymous) (app://sms.gaiamobile.org/common/vendor/sinon/sinon.js:5390:21)
    at (anonymous) (app://sms.gaiamobile.org/views/inbox/test/unit/inbox_test.js:672:7)
    at wrapper (app://sms.gaiamobile.org/common/test/mocha_generators.js:62:19)
    at run (app://sms.gaiamobile.org/common/vendor/mocha/mocha.js:3709:7)
    at runTest (app://sms.gaiamobile.org/common/vendor/mocha/mocha.js:4081:5)
    at (anonymous) (app://sms.gaiamobile.org/common/vendor/mocha/mocha.js:4127:7)
    at next (app://sms.gaiamobile.org/common/vendor/mocha/mocha.js:4007:1)
    at (anonymous) (app://sms.gaiamobile.org/common/vendor/mocha/mocha.js:4016:7)
    at next (app://sms.gaiamobile.org/common/vendor/mocha/mocha.js:3964:1)
    at (anonymous) (app://sms.gaiamobile.org/common/vendor/mocha/mocha.js:3979:7)
    at done (app://sms.gaiamobile.org/common/vendor/mocha/mocha.js:3700:5)
    at (anonymous) (app://sms.gaiamobile.org/common/vendor/mocha/mocha.js:3712:9)
    at (anonymous) (app://sms.gaiamobile.org/common/test/mocha_generators.js:46:20)
    at wrapper (app://sms.gaiamobile.org/common/test/mocha_generators.js:73:15)
    at run (app://sms.gaiamobile.org/common/vendor/mocha/mocha.js:3709:7)
    at next (app://sms.gaiamobile.org/common/vendor/mocha/mocha.js:3973:5)
    at (anonymous) (app://sms.gaiamobile.org/common/vendor/mocha/mocha.js:3984:5)
    at (anonymous) (app://sms.gaiamobile.org/common/vendor/mocha/mocha.js:4932:28)
    at (anonymous) (app://sms.gaiamobile.org/common/vendor/mocha/mocha.js:4929:1)
    at (anonymous) (app://sms.gaiamobile.org/common/vendor/mocha/mocha.js:4918:21)
    at (anonymous) (app://sms.gaiamobile.org/common/vendor/mocha/mocha.js:1:3)`